sys/linux/socket_netlink_route.txt

```diff
@@ -879,6 +882,94 @@ rtcanmsg {
 
 rtcanmsg_flags = CGW_FLAGS_CAN_ECHO, CGW_FLAGS_CAN_SRC_TSTAMP, CGW_FLAGS_CAN_IIF_TX_OK
 
+#TODO: define remaining lwtunnel_encap types
+lwtunnel_policy [
+	MPLS_IPTUNNEL_DST	nlattr[MPLS_IPTUNNEL_DST, array[int8, 4]]
+	MPLS_IPTUNNEL_TTL	nlattr[MPLS_IPTUNNEL_TTL, int8]
+
+	LWTUNNEL_IP_ID		nlattr[LWTUNNEL_IP_ID, int64]
+	LWTUNNEL_IP_DST		nlattr[LWTUNNEL_IP_DST, int32]
+	LWTUNNEL_IP_SRC		nlattr[LWTUNNEL_IP_SRC, int32]
+	LWTUNNEL_IP_TTL		nlattr[LWTUNNEL_IP_TTL, int8]
+	LWTUNNEL_IP_TOS		nlattr[LWTUNNEL_IP_TOS, int8]
+	LWTUNNEL_IP_FLAGS	nlattr[LWTUNNEL_IP_FLAGS, int16]
```

Comment:
`int16be` and one of `TUNNEL_GENEVE_OPT` and others.

Reply:
Fixed.